docs/options.md

```diff
@@ -184,6 +184,61 @@ to limit builds for each job; this list could grow in the future.
 CIBW_ARCHS_LINUX: "auto aarch64"
 ```
 
+
+###  `CIBW_PROJECT_REQUIRES_PYTHON` {: #requires-python}
+> Limit the Python versions to build
```

Comment:
I'm not gonna drop this :) "Limit the Python versions to build" is too close to BUILD/SKIP - that's why I had the 'override compatibility' line in my version.

Reply:
That's historical, sorry, I didn't replace the title but just edited it, so the old one (or slightly modified old one?) stuck.

Currently trying to see why MyPy in pre-commit is not picking up an error in `__main__` for my rewrite. It's working when I run it manually.

Reply:
Aha, pre-commit adds `--ignore-missing-imports` by default.